src/App.svelte

```diff
@@ -61,77 +91,115 @@
 </script>
 
 <svelte:head>
-
+  <link
+  rel="stylesheet"
+  href="https://unpkg.com/carbon-components-svelte/css/white.css"
```

Comment:
Let's not download it from the web inside user's browser. Pretty certian we can host it locally and maybe even rely on npm for refreshing its version automatically.